### ISSUE TITLE
Min Web API: Two problems: ID in PUT body and Put Example returns Status 200 not 204

### ISSUE THREAD

**Issue by jmhmaine** [dotnet/AspNetCore.Docs#32239]
### Description

Update by wadepickett:  The issue brought up in the discussion of the ID being passed in the PUT body is an issue to address.  The 2nd issue of the wrong status code could not be recreated.

2nd part of the issue noted here after it came up in the discussion, which is to be addressed:

"The other item on that page that should be updated is in the **Test the PUT endpoint** section. 

The example provides the ID in the body of the request but it is not used by the logic:

```
Content-Type: application/json

{
  "id": 1,
  "name": "feed fish",
  "isComplete": false
}

```

The **"id": 1,** line should be removed. The ID is pulled from the URI, not the body of the request. "

---  Original descrption from community member ----

The document states that sending the PUT command will return a 204 status code. Instead it returns a 200. 

![image](https://github.com/dotnet/AspNetCore.Docs/assets/26821102/08e4f210-66ef-44b1-8fdf-c9689ef01664)

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/min-web-api.md

### Document ID

ab408035-6313-a183-9940-32de00e0e977

### Article author

@wadepickett

---
[Associated WorkItem - 251080](https://dev.azure.com/msft-skilling/Content/_workitems/edit/251080)

**Comment by wadepickett** [comment]
@jmhmaine Thanks for letting us know!  I will take a look at it.

**Comment by jmhmaine** [comment]
Not a problem, glad to help. 

The other item on that page that should be updated is in the **Test the PUT endpoint** section. 

The example provides the ID in the body of the request but it is not used by the logic:

```
Content-Type: application/json

{
  "id": 1,
  "name": "feed fish",
  "isComplete": false
}

```

The **"id": 1,** line should be removed. The ID is pulled from the URI, not the body of the request.

**Comment by timdeschryver** [comment]
Hi, sorry to come between.
I also tried this, but I wasn't able to reproduce this.
Can you please double check that the `NoContent` result type is returned, as within the example:

```ts
app.MapPut("/todoitems/{id}", async (int id, Todo inputTodo, TodoDb db) =>
{
    var todo = await db.Todos.FindAsync(id);

    if (todo is null) return Results.NotFound();

    todo.Name = inputTodo.Name;
    todo.IsComplete = inputTodo.IsComplete;

    await db.SaveChangesAsync();

    return Results.NoContent();
});
```

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/4e894bce-f73d-41a2-9e33-d2dbaa6db485)

**Comment by wadepickett** [comment]
Thanks @timdeschryver for testing it out and thanks @jmhmaine for letting us know your experience.

@jmhmaine, you are correct about the needless `"id": 1` line for the PUT body.  I see we did remove that mistake in the VS Code version but not in the VS version.  I will get it out of there.  Thanks!

Testing the doc steps I am getting a response of 204 No Content as expected.  I also looked around for anything that might be misinterpreted in the instructions and result in the wrong response as a result.  I did not see anything that stood out.  Below are my results in a screenshot:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/10985336/74a029b5-f68c-4106-bdc3-883c13feee89)
